src/main/java/core/basesyntax/strategy/OperationStrategyImpl.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.exceptions.NotSuchOperationException;
+import core.basesyntax.services.OperationHandler;
+import core.basesyntax.services.impl.AddOperationHandler;
+import core.basesyntax.services.impl.PurchaseOperationHandler;
+import java.util.HashMap;
+import java.util.Map;
+
+public class OperationStrategyImpl implements OperationStrategy {
+    private final Map<String, OperationHandler> operationMap = new HashMap<>();
+
+    public OperationStrategyImpl() {
```

Comment:
you will need to create this map and dao implementation in main - inject dao to handlers, and then pass the map into this constructor